mobile/mysterium/openvpn_connection_setup.go

```diff
@@ -153,8 +153,8 @@ func (c *openvpnConnection) Start(options connection.ConnectOptions) error {
 		return err
 	}
 
-	// TODO this backward compatibility check needs to be removed once we will start using port ranges for all peers.
-	if sessionConfig.LocalPort > 0 || len(sessionConfig.Ports) > 0 {
+	// TODO this backward compatibility block needs to be removed once we will fully migrate to the p2p communication.
+	if len(sessionConfig.Ports) > 0 {
```

Comment:
Mobile openvpn also need to use 2 NAT UDP conn, the same as in wireguard_connection_setup `options.ProviderNATConn`

Reply:
It's done in the `NewClientConfigFromSession` which is shared between mobile and desktop.
https://github.com/mysteriumnetwork/node/pull/1946/files#diff-2a60cf8a8b871ed9f1078ce9e47c33f2R103-R106